docs/creating-new/custom.rst

```diff
@@ -4,6 +4,179 @@
 Non-CMake: custom scheme
 ------------------------
 
-* How to add a new `custom-build project`_
+Non-CMake projects can be added too. But sometimes it's not a trivial task
+(for example there are a 3 custom schemes for `OpenSSL <https://github.com/ruslo/hunter/blob/master/cmake/projects/OpenSSL/hunter.cmake>`_. In general it's better to
+apply a patch to an existing CMake build and use :ref:`cmake-simple` add
+instruction. Anyway here is a guide how to add a project with custom build:
 
-.. _custom-build project: https://github.com/ruslo/hunter/wiki/usr.adding.new.package.custom.scheme
+#. Test it manually
```

Comment:
Fix format

Reply:
This is actually valid. See here: http://www.sphinx-doc.org/en/stable/rest.html#lists-and-quote-like-blocks.

Reply:
Never mind it is not rendering properly.